packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -270,6 +271,16 @@ abstract mixin class WidgetsBindingObserver {
   ///    application lifecycle changes.
   void didChangeAppLifecycleState(AppLifecycleState state) { }
 
+  /// Called whenever the [PlatformDispatcher] receives a notification that the
+  /// focus state on a view has changed.
+  ///
+  /// The [event] contains the view ID for the view that changed its focus
+  /// state.
+  ///
+  /// The view id of the [FlutterView] in which a particular [BuildContext]
+  /// resides can be retrieved from the [ViewFocusEvent].
```

Comment:
The `id` in the event is not necesarily the same as the one of the view the buildcontext is in, though?

Reply:
Right. Fixed, and I added why you might want to look up the ID for a build context.